test/suite-functional/api/functional.link.spec.php

```diff
@@ -161,6 +161,31 @@
 
         $strandA->terminate();
     });
+
+    rit('can link to self', function () {
```

Comment:
It's probably best if the test name describes the feature, so in this case it's not that the strand can link to itself, but moreso that it "does nothing when linking to self".